Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nginx:stable-alpine
+
+RUN mkdir -p /var/cache/nginx/client_temp /var/run/nginx
+
+RUN chown -R nginx:nginx /var/cache/nginx /var/run/nginx /usr/share/nginx/html
+
+COPY ./nginx.conf /etc/nginx/nginx.conf
+
+COPY ./angular/dist/angular-starter/ /usr/share/nginx/html
+
+USER nginx
```

Comment:
Makse condensed. Separate with lines ONLY when you need to emphasize something. Here you emphasize everything, means no emphasis (as everything is equal)